ics5110_ml_handbook.tex

```diff
@@ -167,7 +167,7 @@
 
 \par Copyright \copyright\ \the\year\ \ics\ class of 2018/9, University of Malta.
 
-\par\smallcaps{Jean-Paul Ebejer, Dylan Seychell, \hl{ADD YOUR NAME HERE}} %% TODO
+\par\smallcaps{Jean-Paul Ebejer, Dylan Seychell, \hl{Lara Marie Demajo}}
```

Comment:
Add your name, and leave \hl{ADD YOUR NAME HERE} for others to know what they have to do.